deployments/docker/eos9al-stable.repo

```diff
@@ -1,9 +1,9 @@
 # Added by EOSxd CSI driver.
 # Koji repository with EOS packages.
 
-[eos9-stable]
+[eos9al-stable]
 name=EOS binaries from CERN Koji [stable]
-baseurl=http://linuxsoft.cern.ch/internal/repos/eos9-stable/$basearch/os
+baseurl=http://linuxsoft.cern.ch/internal/repos/eos9al-stable/$basearch/os
```

Comment:
Now that we are in GitHub, maybe it's worth creating a public repo so that externals can build test images?

Reply:
Maybe track that in a separate issue?